sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-storage-resource/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,8 @@
+spring.cloud.azure.credential-file-path=file:@credential@
+spring.cloud.azure.resource-group=spring-cloud
+
+blob=azure-blob://container1/blob1
+spring.cloud.azure.storage.account=springcloud
+
+spring.cloud.azure.region=westUS
+spring.cloud.azure.auto-create-resources=true
```

Comment:
Better to use engineering system's capability to provision Azure resource dynamically. Better not to leave storage URL in config file.